nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.listen(3000, () => console.log(`Calculator:listening on port 3000`));
+
+app.get("/calculator/add", (req, res) =>{
+    const {firstNumber, secondNumber} = req.query;
+    res.send(`${firstNumber} + ${secondNumber} = ${Number(firstNumber) + Number(secondNumber)}`);
+    }
+    );
+
+    app.get("/calculator/multiply", (req, res) =>{
```

Comment:
You're also missing the 2 POST requests:
https://github.com/HackYourFuture-CPH/node.js/blob/main/week3/homework-template/readme.md#getting-data-through-the-request-body-using-post